to-do-app/src/components/ListTitles/index.js

```diff
@@ -25,7 +25,9 @@ class ListTitles extends Component {
       lists = titles.map(
         entry => 
           <ul key={entry}>
-            <li onClick={() => this.props.saveActiveList(this.getActiveList(entry))}>
+            <li
+              className={this.props.activeList && entry === this.props.activeList.title ? "active-list" : "unactive-list"} 
```

Comment:
Is there a particular reason why you're using `entry` instead of` title`? Also is there a test for the classname logic?

Reply:
No, we named it with Pedro. I'm trying to do renaming. I'll change this too.

No, I don't have this test. I'll think how I can test this.